draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -830,6 +832,15 @@ This algorithm chooses to match to the earliest query with the correct Primary a
 
 A FIFO structure is used to hold the Q/R data items during processing.
 
+## Matching Queries and Responses
```

Comment:
This overlaps with what is said in Algorithm Limitations. We should put it all together in a single section. Since the entire algorithm is designed to match queries and responses I suggest you move this text to the Algorithm Limitations section?